modules/01-events.js

```diff
@@ -185,6 +185,20 @@ const Module = new Augur.Module()
       ldsg.channels.cache.get(sf.channels.userupdates).send({ content: `${newUser}: ${newUser.id}`, embeds: [embed] });
     }
   } catch (error) { u.errorHandler(error, `User Update Error: ${u.escapeText(newUser?.username)} (${newUser.id})`); }
+})
+.addEvent("messageCreate", async (msg) => {
+  if (!msg.author.bot && msg.guild && msg.guild.id == sf.ldsg) {
+    let i = 0;
+    do {
+      const [sponsor, emoji] = emojis[i];
+      // Sponsor Pings
+      console.log(!(!isNaN(sponsor) && sponsor.length == 18) && msg.content.toLowerCase().includes(sponsor));
```

Comment:
This will spam console logs and should be removed.

Reply:
Ah, forgot to remove that